ui/raidboss/data/04-sb/trial/seiryu-ex.ts

```diff
@@ -191,17 +209,18 @@ export default {
     },
     {
       id: 'SeiryuEx Ascending Tower You',
+      type: 'HeadMarker',
       netRegex: NetRegexes.headMarker({ id: '00A9', capture: false }),
       condition: (data) => {
-        if (!data.blazing || data.markers.length !== 4)
+        if (!data.blazing || !data.markers || data.markers.length !== 4)
```

Comment:
```suggestion
        if (!data.blazing || data.markers?.length !== 4)
```

Reply:
This is not the same.  I want this to be truthy when `!data.markers`.

Reply:
Oh whoops, you're right, sry!